plugin/pkg/auth/authorizer/node/BUILD

```diff
@@ -35,7 +35,7 @@ go_library(
         "graph.go",
         "graph_populator.go",
         "intset.go",
-        "node_authorizer.go",
+        "authorizer.go",
```

Comment:
please revert these filenames... specific filenames allows using --vmodule to increase/decrease log levels just for this component